compiler/rustc_attr_parsing/src/context.rs

```diff
@@ -555,6 +555,25 @@ pub enum OmitDoc {
     Skip,
 }
 
+#[derive(Copy, Clone)]
+pub enum ShouldEmit {
+    /// The operation will emit errors and lints.
+    /// This is usually what you need.
+    Emit,
```

Comment:
Can we maybe make them even more explicit, since looking at `ShouldEmit::Emit` is still quite unclear at first glance. What about `ShouldEmit::Errors` (or `ShouldEmit::ErrorsAndLints`) and `ShouldEmit::Nothing`.

Reply:
Fixed :)